### PR TITLE
Add Meriones unguiculatus to species list

### PR DESCRIPTION
**Pull request by kabilar**
Reference: http://purl.obolibrary.org/obo/NCBITaxon_10047

### CODE REVIEW

File: dandi/metadata/util.py [hunk 1]
```diff
@@ -394,6 +394,12 @@ def extract_cellLine(metadata: dict) -> str | None:
         NCBITAXON_URI_TEMPLATE.format("9545"),
         "Macaca nemestrina",
     ),
+    (
+        ["Mongolian gerbil", "Mongolian jird"],
```
Comment (jwodder) [inline dandi/metadata/util.py:398]: These need to be lowercase.

**Review by jwodder — CHANGES_REQUESTED** [review]

File: dandi/metadata/util.py (hunk 1 above)
Comment (yarikoptic) [reply dandi/metadata/util.py:398]:
```suggestion
        ["mongolian gerbil", "mongolian jird"],
```

Comment (yarikoptic) [reply dandi/metadata/util.py:398]: ideally IMHO we should make code robust to specification here and just `.lower()` then when matching. Or do you see an issue with that @jwodder ?

**Comment by kabilar** [comment]
Thank you, @yarikoptic @jwodder.  Would you be able to make a release this week so that I can notify the user who requested this change?

**Comment by yarikoptic** [comment]
Sure.
I am looking into hopefully finalize
- https://github.com/dandi/dandi-cli/pull/1536

with which we could trigger release... if not - I will generate empty PR to trigger it

**Pull request merged**